src/reactors/PriorityOrderReactor.sol

```diff
@@ -46,10 +44,11 @@ contract PriorityOrderReactor is BaseReactor {
         _validateOrder(orderHash, order);
 
         uint256 priorityFee = tx.gasprice - block.basefee;
```

Comment:
if tx.gasprice < block.basefee for some reason (maybe weird chain behavior etc) maybe we should either:
- revert with nicer message
- just use priority fee of 0 rather than revert?
Thoughts?

Reply:
hmmm so I thought a bit about this too,
- definitely agree with reverting with a custom error instead of the underflow
- I'm hesitant on doing priority fee of 0 because that basically skips the MEV tax mechanism for these orders, and I'm worried that a chain could change behavior and we wouldn't catch it because we have this silent fallback

Reply:
ok synced - let's just revert with custom error